wazo_auth/services/saml_config.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022-2024 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import os
+from xml.etree import ElementTree
+
+from wazo_auth.database.models import Domain, SAMLConfig
+from wazo_auth.plugins.http.saml_config.schemas import saml_config_schema
+from wazo_auth.services.helpers import BaseService
+
+logger = logging.getLogger(__name__)
+
+
+class SAMLConfigService(BaseService):
+    def __init__(self, config, saml_service, dao) -> None:
+        self._xml_files_dir: str = config['saml']['xml_files_dir']
+        self._saml_service = saml_service
+        super().__init__(dao)
+        self._reload_saml_service()
```

Comment:
I see no reason not to call super first
```suggestion
        super().__init__(dao)
        self._xml_files_dir: str = config['saml']['xml_files_dir']
        self._saml_service = saml_service
        self._reload_saml_service()
```